### DISCUSSION TITLE
Why does Nemo creates the 'Desktop directory if it doesn't exist ?

### DISCUSSION THREAD

**Discussion by Aeldit** [linuxmint/discussions#616] [Questions] [answered]
For context, I am using i3 and thus don't need the Desktop folder. But every time I remove it, Nemo re-creates it. Is there a way to disable this ?

**Comment by mtwebster** [comment, marked as answer]
You can run:
```
gsettings set org.nemo.preferences desktop-is-home-dir true
```

That will prevent it being created. Unfortunately it will still show up in the sidebar (as a broken bookmark). I'll address this for 6.4.

**Reply by Aeldit** [reply]
That worked, thanks :)
